File: src/IsbInterop/BaseIsbObjectImp.vb
''' <summary>
''' Имплементация базового объекта IS-Builder.
''' </summary>
Friend Class BaseIsbObjectImp
  Inherits BaseIsbObject

  ''' <summary>
  ''' Конструктор.
  ''' </summary>
  ''' <param name="rcwObject">COM-объект.</param>
  ''' <param name="scope">Область видимости.</param>
  Public Sub New(rcwObject As Object, scope As IScope)
    MyBase.New(rcwObject, scope)
  End Sub
End Class

File: src/IsbInterop/BaseIsbObjectImp.cs
namespace IsbInterop
{
  /// <summary>
  /// Имплементация базового объекта IS-Builder.
  /// </summary>
  internal sealed class BaseIsbObjectImp : BaseIsbObject
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwObject">COM-объект.</param>
    /// <param name="scope">Область видимости.</param>
    public BaseIsbObjectImp(object rcwObject, IScope scope)
      : base(rcwObject, scope) { }
  }
}
